server/monitor/src/main/resources/resources/master.js

```diff
@@ -0,0 +1,261 @@
+      var items = [];
+      items.push('<td class="firstcell left" data-value="' + val.server + '">' +
+          val.server + '</td>');
+      items.push('<td class="right" data-value="' + val.log + '">' + val.log +
+          '</td>');
+      var date = new Date(parseInt(val.time));
+      date = date.toLocaleString().split(' ').join('&nbsp;');
+      items.push('<td class="right" data-value="' + val.time + '">' + date +
+          '</td>');
+      items.push('<td class="right" data-value="' + val.copySort + '">' +
+          val.copySort + '</td>');
+
+      $('<tr/>', {
+        html: items.join('')
+      }).appendTo('#recoveryList');
+    });
+  }
+}
+
+/**
+ * Generates the master table
+ */
+function refreshMasterTable() {
+  // Gets the master status
+  var status = JSON.parse(sessionStorage.status).masterStatus;
+
+  // Hide the banner and the master table
+  $('#masterBanner').hide();
+  $('#masterStatus tr:gt(0)').remove();
+  $('#masterStatus').hide();
+
+  // If master status is error, show banner, otherwise, create master table
+  if (status === 'ERROR') {
+    $('#masterBanner').show();
+  } else {
+    $('#masterStatus').show();
+    var data = JSON.parse(sessionStorage.master);
+    var items = [];
+    items.push('<td class="firstcell left" data-value="' + data.master +
+        '">' + data.master + '</td>');
+
+    items.push('<td class="right" data-value="' + data.onlineTabletServers +
+        '">' + data.onlineTabletServers + '</td>');
+
+    items.push('<td class="right" data-value="' + data.totalTabletServers +
+        '">' + data.totalTabletServers + '</td>');
+
+    var date = new Date(parseInt(data.lastGC));
+    date = date.toLocaleString().split(' ').join('&nbsp;');
+    items.push('<td class="left" data-value="' + data.lasGC +
+        '"><a href="/gc">' + date + '</a></td>');
+
+    items.push('<td class="right" data-value="' + data.tablets +
+        '">' + bigNumberForQuantity(data.tablets) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.unassignedTablets +
+        '">' + bigNumberForQuantity(data.unassignedTablets) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.numentries +
+        '">' + bigNumberForQuantity(data.numentries) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.ingestrate +
+        '">' + bigNumberForQuantity(Math.round(data.ingestrate)) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.entriesRead +
+        '">' + bigNumberForQuantity(Math.round(data.entriesRead)) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.queryrate +
+        '">' + bigNumberForQuantity(Math.round(data.queryrate)) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.holdTime +
+        '">' + timeDuration(data.holdTime) + '</td>');
+
+    items.push('<td class="right" data-value="' + data.osload +
+        '">' + bigNumberForQuantity(data.osload) + '</td>');
```

Comment:
You should create a function to clean this up.  Something simple like createTableCell(dataValue, prettyValue) would help a lot.